agent/http-server.c

```diff
@@ -189,8 +190,10 @@ http_cb (SoupServer * server, SoupMessage * msg, const char *path,
   soup_buffer_free (buffer);
   soup_message_set_status (msg, SOUP_STATUS_OK);
 
+  params = g_hash_table_new (g_str_hash, g_str_equal);
   file_name = g_path_get_basename (file_path);
-  soup_message_headers_append (msg->response_headers, "filename", file_name);
+  g_hash_table_insert (params, "filename", file_name);
+  soup_message_headers_set_content_disposition (hdrs, "attachment", params);
```

Comment:
There is no hdrs. Is it msg->response_headers instead of hdrs?

Reply:
ah.. typo... I'll fix it.